cmd/cdi-apiserver/apiserver.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	aggregatorclient "k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset"
+
+	"kubevirt.io/containerized-data-importer/pkg/apiserver"
+	. "kubevirt.io/containerized-data-importer/pkg/common"
+)
+
+const (
+	// Default port that api listens on.
+	defaultPort = 8443
+
+	// Default address api listens on.
+	defaultHost = "0.0.0.0"
+)
+
+var (
+	configPath string
+	masterURL  string
+	verbose    string
+)
+
+func init() {
+	// flags
+	flag.StringVar(&configPath, "kubeconfig", os.Getenv("KUBECONFIG"), "(Optional) Overrides $KUBECONFIG")
+	flag.StringVar(&masterURL, "server", "", "(Optional) URL address of a remote api server.  Do not set for local clusters.")
+	flag.Parse()
+
+	// get the verbose level so it can be passed to the importer pod
+	defVerbose := fmt.Sprintf("%d", DEFAULT_VERBOSE) // note flag values are strings
+	verbose = defVerbose
+	// visit actual flags passed in and if passed check -v and set verbose
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == "v" {
+			verbose = f.Value.String()
+		}
+	})
+	if verbose == defVerbose {
```

Comment:
Minor nit but it might be more clear to formally declare the 'v' flag and then just do a check/compare explicitly using strconv.Itoa(v)... 
Had the same thought on the controller but it translates here as well, maybe something like this https://gist.github.com/j-griffith/a8ac7dc5476c453fafc6b31fce9c54c1

It's still ugly to have the extra 'v' variable in there but it makes this pretty clear and reduces the code a bit.  Even better at some point we should probably consider restructuring these shared mains that all have this same base parsing, maybe an external/shared parsing module?

Reply:
That does look nicer!  Yeah, shared main may be the way to go though.